demos/list.html

```diff
@@ -1200,6 +1329,48 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        var deactivateRippleHandler = null;
+        var checkboxListRippleSurfaceClass = 'checkbox-list-ripple-surface';
+
+        function clickHandler(evt, rippleFoundation, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          var isLabel = Array.prototype.some.call(checkbox.labels, function(label) { return label === evt.target; });
+          if(evt.target !== checkbox && !isLabel) {
+            checkbox.checked = !checkbox.checked;
+            li.classList.add(checkboxListRippleSurfaceClass);
+            rippleFoundation.activate();
```

Comment:
If I pass the event here, I don't need the deactivateRippleHandler logic.

Reply:
That might be a better idea cross-device, since that will allow the ripple foundation to register all types of pointer events for deactivation rather than just mouseup like you're doing here.

Alternatively, what form-field does is simply call deactivate on the next animation frame.

Reply:
Yeah, let's pass the event to `activate()` so we don't need the `mouseup` handler.